wagtail_wordpress_import/test/tests/test_inline_shortcode_handlers.py

```diff
@@ -0,0 +1,44 @@
+from re import Match
+from django.test import TestCase
+
+from wagtail_wordpress_import.handle_inline_shortcodes import InlineShortcodeHandler
+
+
+class TestInlineShortcodeHandler(TestCase):
```

Comment:
**Suggestion**: If we have concerns about cases where the regex does or doesn't behave correctly, it would be good to add tests here, even if they're decorated with `@skipif(True, "This is not yet fully featured")` (see https://docs.python.org/3/library/unittest.html#unittest.skipIf). That way at least we're documenting the desired behaviour.

Reply:
I'm not sure if this is needed now. I have added quite a few more tests.

Reply:
Updated